docs/reference/get-testnet-zeta.mdx

```diff
@@ -7,27 +7,33 @@ sidebar_position: 6
 
 ## What is ZETA?
 
-ZETA is a token used to deploy and call smart contracts on ZetaChain, and to transact
-data and value across chains (through messaging). For ZetaChain's Athens
-network, they use the same ZETA that lives on connected testnet networks. With
-ZetaChain's Mainnet launch, ZETA will be traded and transacted across all
+ZETA is a token used to deploy and call smart contracts on ZetaChain, and to
+transact data and value across chains (through messaging). For ZetaChain's
+Athens network, they use the same ZETA that lives on connected testnet networks.
+With ZetaChain's Mainnet launch, ZETA will be traded and transacted across all
 connected chains.
 
 You can view the addresses on respective chains [here](/reference/testnet.mdx).
-Check out [Omnichain Smart Contacts](/developers/overview.mdx) for
-more information about developing smart contracts on ZetaChain, where ZETA is
-used for gas fees. Check out the [Connector
-docs](/developers/cross-chain-messaging/connector.mdx) for information on how to send
-ZETA as well as any data or value via ZetaChain.
+Check out [Omnichain Smart Contacts](/developers/overview.mdx) for more
+information about developing smart contracts on ZetaChain, where ZETA is used
+for gas fees. Check out the
+[Connector docs](/developers/cross-chain-messaging/connector.mdx) for
+information on how to send ZETA as well as any data or value via ZetaChain.
 
 ## How to get testnet ZETA
 
 :::note
+
 Please note that testnet ZETA has no monetary value and is solely intended for
 testing and development purposes. Therefore, if you're not a developer, there's
 no need for you to acquire testnet ZETA.
+
 :::
 
+To obtain testnet ZETA on the current recommended **Athens 2 testnet** for
+development purposes, please follow the instructions provided below. Please note
+that the faucet will soon be updated to support Athens 3 in the coming weeks.
```

Comment:
Maybe we can add one sentence:

If you need to get assets for Athens-3, please ask in our developer channel `dev-general`.

Reply:
We're going to get spammed with requests from non devs. I propose when we merge the Athens 3 PRs, I'll write a short tutorial on how to swap gETH/etc. into ZETA and send ZETA to ZetaChain using `npx hardhat send-zeta`. This is almost as good as a faucet and at least doesn't require one of us to manually send tokens.